app/1.0/docs/devguide/data-binding.md

```diff
@@ -88,412 +123,257 @@ annotation inside the child element.
+### Native properties that don't support property binding {#native-binding}
 
-*   When the property changes, the element fires a non-bubbling DOM
-    event to indicate those changes to interested hosts.
+There are a handful of common native element properties that Polymer can't data-bind to directly,
+because the binding causes issues on one or more browsers.
 
-*   The event type follows a naming convention of <code><var>property</var>-changed</code>,
-    where property's mixedCase words are written with dashes. E.g. a change to
-    `this.firstName` will fire `first-name-changed`. Listeners will receive an event object
-    whose `e.detail.value` attribute has the changing property's new value.
+You need to use attribute bindings to affect the following properties:
 
-You can manually attach a <code><var>property</var>-changed</code>
-listener to an element to notify external elements, frameworks,
-or libraries of property changes.
+| Attribute | Property | Notes
 
-This is essentially what Polymer does when you create a
-two-way data binding.
+| `class` | `classList`, `className` | Maps to two properties with different formats.
 
-### Two-way binding to native elements {#two-way-native}
+| `style` | `style` | By specification, `style` is considered a read-only reference to a `CSSStyleDeclaration` object.
 
-As mentioned above, Polymer uses an event naming convention to achieve two-way
-binding.
+| `href` | `href` |
 
-To two-way bind to native elements or non-Polymer elements that _don't_
-follow this event naming convention, you can specify a
-custom change event name in the annotation using the following syntax:
+| `for` | `htmlFor` |
 
-<code><var>target-prop</var>="{{<var>hostProp</var>::<var>target-change-event</var>}}"</code>
+| `data-*` |  `dataset` | Custom data attributes (attribute names starting with `data-`) are stored on the `dataset` property.
 
+| `value` | `value` | Only for `<input type="number">`.
 
-Example: { .caption }
+**Note:** data binding to the `value` property doesn't work on IE for ***numeric input types***. For
+this specific case, you can use one-way attribute binding to set the `value` of a numeric input. Or
+use another element such as `iron-input` or `paper-input` that handles two-way binding correctly.
+{.alert .alert-info }
 
-```
+This list includes the properties currently known to cause issues with property bindings. Other
+properties may also be affected.
 
-<!-- Listens for `input` event and sets hostValue to <input>.value -->
-<input value="{{hostValue::input}}">
+There are various reasons that properties can't be bound:
 
-<!-- Listens for `change` event and sets hostChecked to <input>.checked -->
-<input type="checkbox" checked="{{hostChecked::change}}">
-
-<!-- Listens for `timeupdate ` event and sets hostTime to <video>.currentTime -->
-<video url="..." current-time="{{hostTime::timeupdate}}">
+*   Cross-browser isssues with the ability to place binding braces `{{...}}` in some of these
```

Comment:
I have never been able to track down what exactly the issues are here. So, not sure what we'd say.